src/wallet/index.ts

```diff
@@ -20,8 +20,8 @@ export class Wallet {
     this._callbacks = _callbacks
   }
 
-  set callbacks(o: WalletActionsCallbacks) {
-    this._callbacks = o
+  public updateCallbacks(callbacks: Partial<WalletActionsCallbacks>) {
```

Comment:
`setCallbacks` would imho be less ambiguous of a name, somebody may misinterpret `updateCallbacks` as a function providing "updates" to the callbacks, i.e. notifying them about some event or something like that, which obviously isn't the case